Resources/Prototypes/Entities/Objects/Misc/land_mine.yml

```diff
@@ -68,9 +82,25 @@
   id: LandMineExplosive
   components:
   - type: ExplodeOnTrigger
+  - type: LandMine
   - type: Explosive
     explosionType: Default
     maxIntensity: 10
     intensitySlope: 3
     totalIntensity: 120 # about a ~4 tile radius
     canCreateVacuum: false
+
+- type: entity
+  name: explosive mine
```

Comment:
remove name line, also taken from the parent

Reply:
Fixed